README.md

```diff
@@ -83,6 +83,10 @@ $ make bootstrap
 ...
 ```
 
+> [!WARNING]
```

Comment:
Would you be ok putting this in a new section above "Importing Vault" like

### Troubleshooting

If you encounter an error like *`could not read Username for 'https://github.com'`* you may need to adjust your git config like so:
```sh
$ git config --global --add url."git@github.com:".insteadOf "https://github.com/"
```

It renders a little weird in the README file right now, and I'm sure there might be more troubleshooting common build issues tips out there.

Reply:
@mladlow a new section is a good idea. I've adjusted as suggested. Thank you for the feedback.